parachains/runtimes/collectives/collectives-polkadot/src/lib.rs

```diff
@@ -293,7 +297,13 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 	fn filter(&self, c: &RuntimeCall) -> bool {
 		match self {
 			ProxyType::Any => true,
-			ProxyType::NonTransfer => !matches!(c, RuntimeCall::Balances { .. }),
+			ProxyType::NonTransfer => !matches!(
+				c,
+				RuntimeCall::Scheduler { .. } |
+					RuntimeCall::AmbassadorCollective { .. } |
+					RuntimeCall::AmbassadorReferenda { .. } |
```

Comment:
Sorry, line of my comment was not clear.
```suggestion
				RuntimeCall::Scheduler { .. } |
```